plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/metrics/OTelMetricsTelemetry.java

```diff
@@ -68,6 +70,28 @@ public Counter createUpDownCounter(String name, String description, String unit)
         return new OTelUpDownCounter(doubleUpDownCounter);
     }
 
+    /**
+     * Creates the Otel Histogram. It created the default version. In {@link org.opensearch.telemetry.tracing.OTelResourceProvider}
+     * we can configure the bucketing strategy through view. It appends the OTelResourceProvider.DYNAMIC_HISTOGRAM_METRIC_NAME_SUFFIX
+     * to the metric name. It's needed to differentiate the {@link Histogram} metrics with explictt bucket or dynamic
+     * buckets. This suffix can be removed from the metric name in the collector.
+     * @param name        name of the histogram.
+     * @param description any description about the metric.
+     * @param unit        unit of the metric.
+     * @return histogram
+     */
+    @Override
+    public Histogram createHistogram(String name, String description, String unit) {
+        String internalMetricName = name + OTelResourceProvider.DYNAMIC_HISTOGRAM_METRIC_NAME_SUFFIX;
```

Comment:
So we don't need to that anymore, right? We only use this kind of histogram (so we can configure `*`) - it is exceptionally confusing behaviour to alter metric names under the hood.

Reply:
yes, my bad. removing this.